src/Hedgehog/Report.fs

```diff
@@ -3,25 +3,28 @@ namespace Hedgehog
 [<Measure>] type tests
 [<Measure>] type discards
 [<Measure>] type shrinks
-
-[<RequireQualifiedAccess>]
-type RecheckType =
-    | None
-    | CSharp
-    | FSharp
-    
     
 [<Struct>]
 type RecheckData = internal {
     Size : Size
     Seed : Seed
 }
 
+[<RequireQualifiedAccess>]
+type RecheckType =
+    | CSharp
+    | FSharp
```

Comment:
With the removal of `None`, should this be `Language`?

Reply:
Nice.  Then I renamed `RecheckTypeData` to `RecheckInfo`.